### ISSUE TITLE
Use of uninitialized value in lc at

### ISSUE THREAD

**Issue by mmattel** [mikaku/Monitorix#114]
Monitorix version 3.8.0

The log of monitorix shows me two issues:

````
Use of uninitialized value in lc at /usr/lib/monitorix/port.pm line 222.
Use of uninitialized value in lc at /usr/lib/monitorix/port.pm line 252.
````

**Comment by mikaku** [comment]
This is already fixed in the closed issue #110.

**Comment by mmattel** [comment]
havn´t seen that, 
great tool !
thx
